metricflow/cli/main.py

```diff
@@ -36,7 +36,7 @@
 from metricflow.configuration.config_builder import YamlTemplateBuilder
 from metricflow.dataflow.dataflow_plan_to_text import dataflow_plan_as_text
 from metricflow.engine.metricflow_engine import MetricFlowQueryRequest, MetricFlowExplainResult, MetricFlowQueryResult
-from metricflow.engine.utils import path_to_models
+from metricflow.engine.utils import model_build_result_from_config, path_to_models
```

Comment:
Your commit message has a typo in it. Github won't let me make that comment any other way so I'm putting it here.

Reply:
Fixed 👍